upgrade-monitoring-services.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Upgrade Cluster Monitoring Services
+summary: Learn how to upgrade the Prometheus, Grafana, and Alertmanager monitoring services for your TiDB cluster.
+---
+
+# Upgrade TiDB Cluster Monitoring Services
+
+When you deploy a TiDB cluster using TiUP, TiUP automatically deploys monitoring services (such as Prometheus, Grafana, and Alertmanager) for the cluster. If you scale out this cluster, TiUP also automatically adds monitoring configurations for newly added nodes during the scaling.
+
+Pay attention that TiUP uses its own configurations to override the configurations of the monitoring services. If you directly upgrade the monitoring services by replacing their configuration files, any subsequent operations such as `deploy`, `scale-out`, `scale-in`, or `reload` on the cluster might overwrite your upgrade, leading to errors. To upgrade Prometheus, Grafana, and Alertmanager, you can follow the upgrade steps in this document.
+
+> **Note:**
+>
+> - If your monitoring services are not deployed and managed by TiUP, you can directly upgrade them without referring to this document.
```

Comment:
Can we add a link to the guide that describes how to configure a TiDB cluster to use an external Grafana/Prometheus setup?

Reply:
Do you refer to this one?
https://docs.pingcap.com/tidb/stable/deploy-monitoring-services

Reply:
Yes

Reply:
added in https://github.com/pingcap/docs/pull/14523/commits/2f3c904cf80016662dd4a620821caed987fdf20c